ccpage171.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Add an attribute called number_served with a default value of 0.
+# Create an instance called restaurant from this class.
+# Add a method called set_number_served() that lets you set the number of customers that have been served.
+# Add a method called increment_number_served() that lets you increment the number of customers who’ve been served.
+
+class Restaurant():
+    """An attempt to model a restaurant"""
+    def __init__(self, restuaruant_name, restaurant_cuisine):
```

Comment:
To add another attribute, you should pass in `number_served` to the `__init__` method. Otherwise, it will always be 5.

```
def __init__(self, restuaruant_name, restaurant_cuisine, number_served=0):
```